beets/ui/__init__.py

```diff
@@ -1206,11 +1206,30 @@ def _configure(options):
               util.displayable_path(config.config_dir()))
     return config
 
+# Check whether parental directories exist.
+
+
+def database_dir_creation(path):
+    # Ask the user for a choice.
+    return input_yn("The database directory {} does not \
+                       exists. Create it (Y/n)?"
+                    .format(util.displayable_path(path)))
```

Comment:
I suggest just "inlining" this code, since it's pretty simple and not called anywhere else. (That is, just do this prompting right inside the function below instead of having a separate function.)